test/integration/util/teardown-cluster.sh

```diff
@@ -1,5 +1,26 @@
 #!/bin/bash
 
+kubectl config set-context --current --namespace kong
+
+echo -e "\n--------------------- kubectl get all -A ---------------------\n"
+kubectl get all -A
+echo -e "\n--------------------------------------------------------------\n"
+
+echo -e "\n---------- kubectl describe deployment ingress-kong ----------\n"
+kubectl describe deployment ingress-kong
+echo -e "\n--------------------------------------------------------------\n"
+
+for POD in $(kubectl get pods -o=go-template='{{range .items}}{{.metadata.name}}{{end}}')
+do
+    echo -e "\n----------------- kubectl describe pod ${POD} ----------------\n"
+    kubectl describe pod ${POD}
+    echo -e "\n------------ kubectl logs ${POD} ingress-controller ----------\n"
+    kubectl logs ${POD} ingress-controller
+    echo -e "\n------------------ kubectl logs ${POD} proxy -----------------\n"
+    kubectl logs ${POD} proxy
+    echo -e "\n--------------------------------------------------------------\n"
+done
+
```

Comment:
I was having a CI issue and didn't feel the output was verbose enough so I added this.

I would like to keep it permanently if there are no objections, here's example output:

https://github.com/Kong/kubernetes-ingress-controller/runs/2041991536?check_suite_focus=true#step:5:443